packages/components/src/Tabs/tests/chromatic/Tabs.stories.tsx

```diff
@@ -0,0 +1,286 @@
+    }
+} satisfies Story;
+
+export const Mobile = {
+    ...Default,
+    decorators: [
+        Story => (
+            <div style={{ maxWidth: "360px" }}>
+                <Story />
+            </div>
+        )
+    ]
+} satisfies Story;
+
+export const WithIcon = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <SparklesIcon />
+                    <Text>Red Eye Frog</Text>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args
+    }
+} satisfies Story;
+
+export const WithBadge = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <Text>Red Eye Frog</Text>
+                    <Badge>100</Badge>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args
+    }
+} satisfies Story;
+
+export const WithTag = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <Text>Red Eye Frog</Text>
+                    <Tag>New</Tag>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args
+    }
+} satisfies Story;
+
+export const WithAll = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <SparklesIcon />
+                    <Text>Red Eye Frog</Text>
+                    <Badge>100</Badge>
+                    <Tag>New</Tag>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args
+    }
+} satisfies Story;
+
+export const WithMediumIcon = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <SparklesIcon />
+                    <Text>Red Eye Frog</Text>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args,
+        size: "md"
+    }
+} satisfies Story;
+
+export const WithMediumBadge = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <Text>Red Eye Frog</Text>
+                    <Badge>100</Badge>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args,
+        size: "md"
+    }
+} satisfies Story;
+
+export const WithMediumTag = {
+    render: args => (
+        <Tabs {...args} >
+            <TabList>
+                <Tab id="red-eye-tree">
+                    <Text>Red Eye Frog</Text>
+                    <Tag>New</Tag>
+                </Tab>
+            </TabList>
+        </Tabs>
+    ),
+    args: {
+        ...Default.args,
+        size: "md"
+    }
+} satisfies Story;
+
+export const WithMediumAll = {
```

Comment:
Can you add an example with href="www.google.com" on headers, just so we see that the styling of those headers are ok and not underlined